<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1345</width>
    <height>921</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>1300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(139, 139, 139)</string>
  </property>
  <widget class="QLabel" name="label2_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>451</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>27</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{background-color:rgb(37, 37, 37)}</string>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'华文行楷'; font-size:48pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft YaHei UI'; font-size:36pt; font-weight:700; color:#ffffff;&quot;&gt;视觉引导系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>601</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox::title{
font-size: 72px;
font-weight: bold;
color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="title">
    <string>数据库查询</string>
   </property>
   <widget class="QPushButton" name="disconnect">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(255, 255, 255)；
}</string>
    </property>
    <property name="text">
     <string>断开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(255, 255, 255)；
}</string>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="query">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(255, 255, 255)；
}</string>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QTextEdit" name="restatus">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>351</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>140</y>
     <width>551</width>
     <height>761</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox::title{
font-size: 72px;
font-weight: bold;
color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="title">
    <string>视觉识别</string>
   </property>
   <widget class="QPushButton" name="bt_close">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(255, 255, 255)；
}</string>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_start">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(255, 255, 255)；
}</string>
    </property>
    <property name="text">
     <string>开启</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_show">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>416</width>
      <height>416</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>416</width>
      <height>416</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>416</width>
      <height>416</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>570</y>
     <width>601</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox::title{
font-size: 72px;
font-weight: bold;
color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="title">
    <string>控制指令</string>
   </property>
   <widget class="QPushButton" name="btQscram">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(255, 255, 255)；
}</string>
    </property>
    <property name="text">
     <string>取工具急停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btFscram">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(255, 255, 255)；
}</string>
    </property>
    <property name="text">
     <string>放工具急停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btResetscram">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(255, 255, 255)；
}</string>
    </property>
    <property name="text">
     <string>复位急停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btSetscram">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(255, 255, 255)；
}</string>
    </property>
    <property name="text">
     <string>置位急停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btZscram">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(255, 255, 255)；
}</string>
    </property>
    <property name="text">
     <string>总急停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btrelZscram">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(255, 255, 255)；
}</string>
    </property>
    <property name="text">
     <string>总急停释放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btQ">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(255, 255, 255)；
}</string>
    </property>
    <property name="text">
     <string>取工具</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btF">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(255, 255, 255)；
}</string>
    </property>
    <property name="text">
     <string>放工具</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btSet">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(255, 255, 255)；
}</string>
    </property>
    <property name="text">
     <string>置位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btReset">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(255, 255, 255)；
}</string>
    </property>
    <property name="text">
     <string>复位</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
